### PR TITLE
Get targets when login

### PR DESCRIPTION
**Pull request by brunomichetti**
Screen Shot 2020-07-21 at 10 16 20

- Get targets on login.

### CODE REVIEW

File: src/components/targets-map/MarkerTargets.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import {
+  safron,
+  createTargetOpacity,
+} from '../../style/common/_constants.scss';
+import { getIconByTopic } from './map-icons';
+
+const MarkerTarget = ({ targets }) => (
+  <div>
+    {targets.map(({ _, topic, radius, lat, lon }) => (
+      <div>
```
Comment (manema) [inline src/components/targets-map/MarkerTargets.js]: missing key attribute in div and prop type for targets.

**Review by manema — APPROVED** [review]

File: src/components/targets-map/MarkerTargets.js (hunk 1 above)
Comment (brunomichetti) [reply src/components/targets-map/MarkerTargets.js]: What do you mean?

Comment (manema) [reply src/components/targets-map/MarkerTargets.js]: https://en.reactjs.org/docs/lists-and-keys.html take a look at _keys_ section. As for prop types I mean add MarkerTarget.propTypes = { targets: array }. Do you remember that? I guess we discussed about it

**Review by tamarasuarezrod — APPROVED** [review]
just because in the next pr you saved the targets in the reducer 😆

File: src/components/targets-map/MarkerTargets.js (hunk 1 above)
Comment (brunomichetti) [reply src/components/targets-map/MarkerTargets.js]: Oh thanks! Yes I'll add the propTypes.
About the keys, is it wrong to use `map` in this way: `targets.map(({ _, topic, radius, lat, lon })`?

**Pull request merged**